packages/dara-components/changelog.md

```diff
@@ -2,6 +2,10 @@
 title: Changelog
 ---
 
+## NEXT
+
+-   Fix type of `default_legends` in `CausalGraphViewer` and `VisualEdgeEncoder` to reflect that it does not accept `None`. 
```

Comment:
This PR seems like a good place to also update the docs about `default_legends`?

Reply:
Good point have updated it here